### ISSUE TITLE
Terraform unable to find the AMI Centos 7 in AWS Market-place

### ISSUE THREAD

**Issue by yjash0801** [scholzj/terraform-aws-minikube#14]
Error: Your query returned no results. Please change your search criteria and try again.
│
│   with module.minikube.data.aws_ami.centos7,
│   on .terraform\modules\minikube\main.tf line 131, in data "aws_ami" "centos7":
│  131: data "aws_ami" "centos7" {

**Comment by scholzj** [comment]
Yeah, that is because CentOS 7 was removed. I might get to fix it, but not sure when I will have time for it.

**Comment by yjash0801** [comment]
Is there any alternative AMI I can use to replace the actual AMI in your module?

**Comment by scholzj** [comment]
I don't know. I will likely replace it with Fedora, Amazon Linux or something like that.

**Comment by yjash0801** [comment]
Yeah, thank you can please try to replace it Centos 7 or higher if possible.

**Comment by praveen18k** [comment]
Yeah facing same issue

**Comment by mahussain01** [comment]
yeah facing same issue, even
![Screenshot (591)](https://github.com/user-attachments/assets/c2c1bd0a-d105-45bc-a02f-9cfefd831fa2)
 in centos 8 also. init-aws-minikube.sh is not properly executed in the centos 8.kindly look at the screenshot for not running after the selinux command

**Comment by nishalk09399** [comment]
Getting error like below :

Please do change in the module. However centos7 is not supported now. Please take your time and do work on it. More helpful for us.

Error: Your query returned no results. Please change your search criteria and try again.
│
│ with module.minikube.data.aws_ami.centos7,
│ on .terraform\modules\minikube\main.tf line 131, in data "aws_ami" "centos7":
│ 131: data "aws_ami" "centos7" {

**Comment by scholzj** [comment]
Sorry, I did not have time for this lately. I now updated the project to use CentOS 10.
